tools/untranslated.xml

```diff
@@ -2,14 +2,11 @@
 <resources>
 
     <!-- iOS specific untranslated strings -->
-    <string name="hide">Hide</string>
-    <string name="start_chat">Start Chat</string>
+    <string name="start_chat">Start chat</string>
```

Comment:
We could move that to the android strings.xml (in addition)

Reply:
yip, but let's do that at some point in another pr. the untranslated.xml has always be shrinking the last days, btw, so it is fine it was not done yet :)